orcid-testutils/src/main/java/no/sikt/nva/orcid/testutils/service/OrcidLocalTest.java

```diff
@@ -0,0 +1,73 @@
+package no.sikt.nva.orcid.testutils.service;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
```

Comment:
Class more or less copied from ResourceLocaleTest from publication-api

Reply:
One notable difference: instead of resolving the table name from environment variable, it has to be passed into the init method.